<launch>
	<param name="/use_sim_time" value="true"/>


	<!-- Run the map server -->
  	<node name="map_server" pkg="map_server" type="map_server" args="$(find warehousetest)/maps/warehouse.yaml">
    	<param name="frame_id" value="/map" />
  	</node>


  	<!-- Launch navigation stack for robot1 -->
  	<group ns="robot1">
    	<param name="tf_prefix" value="robot1_tf" />
    	<param name="amcl/initial_pose_x" value="10" />
    	<param name="amcl/initial_pose_y" value="10" />
    	<include file="$(find warehousetest)/config/navigation/move_base.launch.xml" >
    		<arg name="robot_name" value="robot1" />
    	</include>
  	</group>
  	<!-- Launch navigation stack for robot2 -->
  	<group ns="robot2">
    	<param name="tf_prefix" value="robot2_tf" />
    	<param name="amcl/initial_pose_x" value="-10" />
    	<param name="amcl/initial_pose_y" value="-10" />
    	<include file="$(find warehousetest)/config/navigation/move_base.launch.xml" >
    		<arg name="robot_name" value="robot2" />
    	</include>    	
  	</group>






</launch>